hbt/config/variables.py

```diff
@@ -82,42 +81,44 @@ def add_variables(config: od.Config) -> None:
         config,
         name="jet1_eta",
         expression="Jet.eta[:,0]",
-        null_value=EMPTY_FLOAT,
         binning=(30, -3.0, 3.0),
         x_title=r"Jet 1 $\eta$",
     )
+    add_variable(
+        config,
+        name="jet1_phi",
+        expression="Jet.phi[:,0]",
+        binning=(33, -3.3, 3.3),
+        x_title=r"Jet 1 $\phi$",
```

Comment:
Jet 1 or Leading Jet? If to be changed, several occurences below exist too

Reply:
Leading jet. We wanted to convert numbers 1 and 2 to Leader and Subleading
```suggestion
        x_title=r"Subleading jet $\phi$",
```